### PR TITLE
add docs describing how to address drive items in a drive by path / id

### PR DESCRIPTION
**Pull request by zachzi**

### CODE REVIEW

**Review by k-tsoi — APPROVED** [review]

File: api-reference/beta/resources/driveitem.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ There are two primary ways of addressing a **driveItem** resource:
 * By the **driveItem** unique identifier using `drive/items/{item-id}`
 * By file system path using `/drive/root:/path/to/file`
 
+For more information, see [addressing driveItems](../../../concepts/onedrive-addressing-driveitems.md).
```
Comment (k-tsoi) [inline api-reference/beta/resources/driveitem.md:25]: @isvargasmsft how do we address files in the Concepts folder?
The build is failing for this

**Review by isvargasmsft — APPROVED** [review]

**Comment by isvargasmsft** [comment]
@k-tsoi, thank you very much for your work in this PR. 

When the updates in this PR are ready for external customers to use, please replace the **do not merge** label with **ready to merge**.

**Comment by isvargasmsft** [comment]
@k-tsoi, whats the status of this PR? I see its been approved but you haven't added the **ready to merge** label. 

Are you still waiting for the schema to go live?

**Comment by k-tsoi** [comment]
@isvargasmsft The PR is ready to go in, I was unfortunately not tracking this PR very closely.
Thanks for your help with this !!

**Pull request merged**